upup/pkg/kutil/drain.go

```diff
@@ -0,0 +1,674 @@
+/*
+Copyright 2016 The Kubernetes Authors.
```

Comment:
Can you paste the issues for the kubectl bugs?  I struggle to understand why we would do this as anything other than a short-lived fork, until our fixes made it upstream.